perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -52,11 +62,23 @@ echo checkout PR branch
 git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
 echo Mounting gcs bucket from pr branch
 mkdir -p gcs
+BUCKET_NAME=presubmit-perf-test
 # The VM will itself exit if the gcsfuse mount fails.
 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 # Running FIO test
 chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+sudo umount gcs
+
+# Executing implicitdir integration tests for the testbucket flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --testbucket=gcsfuse-implicitdir-tests
+# Mounting gcsfuse for running integration tests for the mountedDirectory flag
+BUCKET_NAME=gcsfuse-implicitdir-tests
+# The VM will itself exit if the gcsfuse mount fails.
+go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+# Executing implicitdir integration tests for the mountedDirectory flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --mountedDirectory=../../../$MOUNT_POINT
```

Comment:
why are we running only implicit dir tests?

Reply:
because for readonly tests we will have different bucket to tests.

Reply:
We need to run all integrations tests as part of daily builds and perf_tests. what is the purpose of adding tests and not running them?